app/assets/javascripts/application.js

```diff
@@ -10,8 +10,9 @@
 // Read Sprockets README (https://github.com/rails/sprockets#sprockets-directives) for details
 // about supported directives.
 //
-//= require rails-ujs
+//= require jquery3
+//= require popper
+//= require bootstrap-sprockets
```

Comment:
We weren't requiring these before, which is why the hamburger nav wasn't working